rate-providers/registry.json

```
@@ -150,5 +150,24 @@
                 }
             ]
         }
+    },
+    "gnosis": {
+        "0xff315299C4d3FB984b67e31F028724b6a9aEb077": {
+            "name": "ERC4626RateProvider",
+            "summary": "safe",
+            "review": "./stAgEurRateProvider.md",
+            "warnings": ["donation", "decimals"],
```

I'd suggest two small changes here:
1. Break the array into multiple lines
2. Call the warning `only18decimals` to be a bit clearer about the specific risk
```suggestion
            "warnings": [
                "donation",
                "only18decimals"
            ],
```